Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -844,8 +898,8 @@
   "block.biomeswevegone.white_mangrove_button": "Кнопка из белого мангрового дерева",
```

Лучше убрать везде лишнее слово "дерева" и заменить на упрощённое "мангр"
```suggestion
  "block.biomeswevegone.white_mangrove_button": "Кнопка из белого мангра",
```